dukes-liberty/src/main/java/eu/agilejava/dukes/HealthEndpoint.java

```diff
@@ -0,0 +1,24 @@
+package eu.agilejava.dukes;
```

Comment:
This class should be removed as it is an old hardcoded health check that is replaced by the MP HealthCheck in the class DukesHealth

Reply:
Nice catch!  I'll remove it shortly...